build.gradle

```diff
@@ -182,12 +182,14 @@ subprojects {
             bouncycastle: 'org.bouncycastle:bcpkix-jdk15on:1.67',
 
             // Test dependencies.
-            junit: 'junit:junit:4.12',
+            junit: 'junit:junit:4.13',
```

Comment:
Internally we're still at 4.12, so we'll probably need this to stay at 4.12 for a while longer.

Reply:
No problem, this was easy to change, reverted.